tasks/sync.js

```diff
@@ -9,14 +9,6 @@
 
 const fs = require('fs-extra');
 const path = require('path');
-const prettier = require('prettier');
-const lerna = require('../lerna.json');
-const packageJson = require('../package.json');
-
-const prettierOptions = {
-  ...packageJson.prettier,
-  parser: 'markdown',
-};
```

Comment:
Are these changes intentional?

Reply:
<img width="1126" alt="Screenshot 2023-05-09 at 19 00 19" src="https://github.com/carbon-design-system/carbon/assets/8973672/4cadd0f6-a632-436d-b562-7ff6a2413404">

Reply:
Okay, I was just unsure why this was being pulled in to this PR. Thanks for clarifying.

Reply:
Agree, this seems unrelated to the issue/PR